lib/iris/unit.py

```diff
@@ -1823,9 +1823,9 @@ def convert(self, value, other, ctype=FLOAT64):
             >>> import numpy as np
             >>> c = unit.Unit('deg_c')
             >>> f = unit.Unit('deg_f')
-            >>> print(c.convert(0, f))
+            >>> print(np.round(c.convert(0, f), 8))
```

Comment:
Really, I just ran a little test and it looks like it prints `32.0` on Python 2 and `31.999999999999886` on Python 3. I could be doing something differently though I suppose. If I'm wrong and it does print `31.999999999999886` on both Python 2 and 3 can we just live with that instead of rounding? Failing that I'd suggest a string formatting instead of rounding, which although not ideal, is more clear what transformation is happening to the resulting data (i.e. showing 1 decimal place rather than rounding which might lose information).